data/ecosystems/s/stellar.toml

```diff
@@ -18,23 +18,40 @@ sub_ecosystems = [
 
 github_organizations = [
   "https://github.com/anclap",
+  "https://github.com/AssetDesk",
   "https://github.com/aurora-rs",
   "https://github.com/blend-capital",
   "https://github.com/blocktimefinancial",
+  "https://github.com/bp-ventures",
+  "https://github.com/CometDEX",
   "https://github.com/CommuniDAO",
   "https://github.com/Creit-Tech",
   "https://github.com/fairxio",
+  "https://github.com/fluxitystellar",
+  "https://github.com/GladiusClub",
```

Comment:
Let's move these github orgs to sub ecosystems since it is more flexible in the future especially if the orgs work in multiple ecosystems.

Reply:
@jubos I can do that, for sure! Couple questions first:

- Are there some "guidelines" you all like use to determine if something should be a sub-ecosystem, as opposed to just a `github_organization`?
- Should I move _all_ of the github orgs I've got in this PR, or just some of them?
- More generally speaking, does EC prefer to have sub-ecosystems set up rather than github orgs?